Microsoft.Toolkit.Uwp.UI.Controls.Media/Microsoft.Toolkit.Uwp.UI.Controls.Media.csproj

```diff
@@ -17,32 +21,21 @@
       Image;Cropper  ;ImageCropper  ;Crop;
       Infinite;Canvas;InfiniteCanvas;
     </PackageTags>
-    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.UI.Xaml" Version="2.5.0" />
-    <PackageReference Include="System.Text.Json" Version="5.0.2" />
-    <PackageReference Include="Win2D.uwp" Version="1.25.0" />
+    <Compile Include="..\Microsoft.Toolkit.Uwp.UI.Controls.Core\ControlHelpers.XamlHost.cs" Link="ControlHelpers.XamlHost.cs" />
+    <None Include="VisualStudioToolsManifest.xml" Pack="true" PackagePath="tools" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Toolkit.Uwp.UI\Microsoft.Toolkit.Uwp.UI.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\Microsoft.Toolkit.Uwp.UI.Controls.Core\ControlHelpers.XamlHost.cs" Link="ControlHelpers.XamlHost.cs" />
-    <None Include="VisualStudioToolsManifest.xml" Pack="true" PackagePath="tools" />
-    <None Include="$(OutDir)\Design\$(MSBuildProjectName).Design*.dll;$(OutDir)\Design\$(MSBuildProjectName).Design*.pdb" Pack="true" PackagePath="lib\$(TargetFramework)\Design" />
```

Comment:
Oh, interesting. This was a mistake before then. @michael-hawker should we open an issue to add a designer project to the Controls.Media package, or is this being already covered by another PR?

Reply:
Will be covered in an upcoming PR.

Reply:
I could either…

- Remove the workaround and add an empty project in this PR, code in the next PR.
- Keep the workaround in this PR. Revert it and add both the project and code in the next PR.

Reply:
@Nirmal4G let's add the Media controls to the Toolbox as part of the #4070 PR.

Reply:
I've already added an empty project in #3794 and started working on it.